tests/test_chrono.py

```diff
@@ -81,7 +81,14 @@ def test_chrono_system_clock_roundtrip_date():
     datetime.time(5, 59, 59, 0),
     datetime.time(5, 59, 59, 1),
 ])
-def test_chrono_system_clock_roundtrip_time(time1):
+@pytest.mark.parameterize("tz", [
+    "/usr/share/zoneinfo/Europe/Brussels",
+    "/usr/share/zoneinfo/Asia/Pyongyang",
+    "/usr/share/zoneinfo/America/New_York",
+])
```

Comment:
Windows won't be a fan of this, though? Though I guess it won't hurt either...

Reply:
It shouldn't really do anything on Windows, I would expect.

Reply:
Except for fooling us (and others) into thinking it ís tested, and running the same tests 3 times. But yes, OK.

We could add `pytest.param(..., marks=pytest.mark.skipif("...", reason="TZ environment variable not supported on Windows")),`?